pkgs/applications/version-management/guilt/default.nix

```diff
@@ -0,0 +1,94 @@
+{ asciidoc
+, docbook_xml_dtd_45
+, docbook_xsl
+, fetchFromGitHub
+, gawk
+, git
+, gnused
+, lib
+, makeWrapper
+, perl
+, stdenv
+, xmlto
+}:
+
+let
+  version = "0.37-rc1";
+  sha256 = "sha256-7OgRbMGYWtGvrZxKfJe0CkpmU3AUkPebF5NyTsfXeGA=";
```

Comment:
Please put these into the expression. There's no need here to put these in a `let`!

Reply:
Done!